src/api/app/controllers/webui/request_controller.rb

```diff
@@ -569,4 +571,10 @@ def prepare_request_data
 
     handle_notification
   end
+
+  def set_influxdb_data_request_actions
+    InfluxDB::Rails.current.tags = {
```

Comment:
This will overwrite all other tags no?

Reply:
No, it is just adding a new tag

Reply:
ok, guess I don't remember my own code then ;)